File: src/main/java/no/nav/syfo/model/Stilling.java
package no.nav.syfo.model;

import lombok.Data;
import lombok.experimental.Accessors;

import java.math.BigDecimal;
import java.time.LocalDate;

@Data
@Accessors(fluent = true)
public class Stilling {
    public String yrke;
    public BigDecimal prosent;
    public LocalDate fom;
    public LocalDate tom;
    public String orgnummer;
}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/oppfolgingsplan/Stilling.kt
package no.nav.syfo.api.v3.domain.oppfolgingsplan

import java.math.BigDecimal
import java.time.LocalDate

data class Stilling(
    var virksomhetsnummer: String,
    var yrke: String,
    var prosent: BigDecimal,
    var fom: LocalDate,
    var tom: LocalDate,
)
